content/en/docs/installation/supported-releases.md

```diff
@@ -183,15 +183,15 @@ The list of supported Kubernetes versions displayed in the [Supported
 Releases](#supported-releases) section depends on what the cert-manager
 maintainers think is reasonable to support and to test.
 
-Our testing coverage is:
-
-| Release branch | Prow configuration            | Dashboard                 | Kubernetes versions tested  |  Periodicity  |
-| :------------: | :---------------------------- | :------------------------ | :-------------------------: | :-----------: |
-|      PRs       | [`presubmits.yaml`][]         | [`presubmits-blocking`][] |            1.22             |  On each PR   |
-|     master     | [`periodics.yaml`][]          | [`master`][]              |         1.16 → 1.22         | Every 2 hours |
-|  release-1.6   | [`next-periodics.yaml`][]     | [`next`][]                |         1.16 → 1.22         | Every 2 hours |
-|  release-1.5   | [`previous-periodics.yaml`][] | [`previous`][]            |         1.16 → 1.22         | Every 2 hours |
-|  release-1.4   | N/A                           |                           |             N/A             |      N/A      |
+As of 16 Dec 2021, Our testing coverage\* is:
+
+| Release branch |      Prow configuration       |         Dashboard         | Kubernetes versions tested |  Periodicity  |
+|:--------------:|:------------------------------|:--------------------------|:--------------------------:|:-------------:|
+|      PRs       | [`presubmits.yaml`][]         | [`presubmits-blocking`][] |            1.22            |  On each PR   |
+|     master     | [`periodics.yaml`][]          | [`master`][]              |        1.18 → 1.23         | Every 2 hours |
+|  release-1.7   | [`next-periodics.yaml`][]     | [`next`][]                |        1.18 → 1.23         | Every 2 hours |
```

Comment:
suggestion: We're not currently running any tests against release-next because we've not cut an alpha yet and so release-1.7 isn't being fast-forwarded. That was done in https://github.com/jetstack/testing/pull/606 - as such, `next-periodics.yaml` is currently a 404; those periodics will return when we cut an alpha.

It might be worth removing the release-1.7 line entirely (and the `next-periodics.yaml` link on line 198) for now? We can add back in when we re-enable the periodics